cli/actions/setup.js

```diff
@@ -234,15 +220,15 @@ module.exports = (config, program) => {
 
     const copyConfig = () => {
       shell.cp(newConfig, userKytConfigPath);
-      logger.task('Created new kyt.config.js');
+      logger.task(`Created ${configFileName} file`);
     };
 
     if (shell.test('-f', userKytConfigPath)) {
       // Since the user already has a kyt.config,
       // we need to back it up before copying.
-      const mvTo = path.join(userRootPath, `kyt.config-${Date.now()}.js.bak`);
+      const mvTo = path.join(userRootPath, `${configFileName}-${Date.now()}.bak`);
```

Comment:
It might make sense to add a const at the very beginning for `Date.now()` so that the times are the exact same for all the backups.